Engine/Utilities.cs

```diff
@@ -253,10 +253,22 @@ public void Dispose()
     }
     public class Texture : IDisposable
     {
+        private interface IResourceViewCollection { }
+        private struct ResourceViewCollection<T> : IResourceViewCollection where T : ResourceView
+        {
+            public struct ArrayItemView
+            {
+                public List<T> MipsViews;
+            }
+
+            public T GeneralView;
+            public List<ArrayItemView> ArrayItemsViews;
```

Comment:
view for an array slice is missing (1 array element, all mips)

Reply:
Fix